Main.java

```diff
@@ -667,10 +679,15 @@ private static void finishFight(Client winner, Client loser) {
       winner.giveItem(Game.Item.HPOTION);
       msg(winner, "You found 1 healing potion!");
     } else {
-      int rnd = Utils.rndInRange(1,4);
+      // logic for looting bots is here
+      int rnd = Utils.rndInRange(1,6);
       if (rnd == 1) {
         winner.giveItem(Game.Item.HPOTION);
         msg(winner, "You found 1 healing potion!");
+      } else if (rnd < 4) {
+        Game.Item found = Game.ITEM_VALUES[Utils.getRnd(loser.inventory)];
+        winner.giveItem(found);
```

Comment:
Am I missing it or there is no place when item actually disappears from loser's inventory?

Reply:
Yes, as of now it doesn't disappear because it only happens with bots and we never reuse the bots who lost. Instead we replace them with new bots when players want to fight next time. Do you think we should remove the item regardless?